metadata/trackers.json

```diff
@@ -23,6 +23,7 @@
         "redact" : {
             "classification:Security": true,
             "tentpole:.*Security.*": true
-        }
+        },
+        "redact_exemption" : {"keywords:[^:]*WebKit Cleared for Publication" : true}
```

Comment:
If we can tie this keyword to the condition it hopes to express, then we can reduce the chances that someone will apply it blindly just to merge their patch. How about:

WebKit Security Does Not Repro in Shipping

Is there any condition where a Security bug does repro in shipping, but we want to support publishing it anyway?

Reply:
We would need to use this same mechanism after shipping security fixes too during merge-back, because all merge-back commits will be referencing security commits which did reproduce in the latest shipping release when the bug was first filed.

We can also support multiple keywords here, so could have multiple supported keywords.